test/AllowIntrinsics.ll

```diff
@@ -0,0 +1,37 @@
+; The test checks command-line option for the translator
+; which will allow to represent unknown llvm intrinsics as external function call in SPIR-V.
+; RUN: llvm-as %s -o %t.bc
+; RUN: llvm-spirv -spirv-allow-unknown-intrinsics %t.bc -o %t.spv
```

Comment:
Maybe add a `spirv-val` step as well, to ensure the generated SPIR-V is valid?

Reply:
Done, thanks for suggestion